src/components/WorkspaceControlPanel.js

```diff
@@ -34,34 +44,73 @@ class WorkspaceControlPanel extends Component {
     });
   }
 
+  /**
+   * @private
+   */
+  handleClick(event) {
+    this.setState({
+      anchorEl: event.currentTarget,
+    });
+  }
+
+  /**
+   * @private
+   */
+  handleClose() {
+    this.setState({
+      anchorEl: null,
+    });
+  }
+
   /**
    * render
    * @return {String} - HTML markup for the component
    */
   render() {
-    const { manifests } = this.props;
-    const { lastRequested } = this.state;
+    const { manifests, classes } = this.props;
+    const { lastRequested, anchorEl } = this.state;
     const manifestList = Object.keys(manifests).map(manifest => (
       <ConnectedManifestListItem
         key={manifest}
         manifest={manifest}
+        handleClose={this.handleClose}
       />
     ));
     return (
-      <div className={ns('workspace-control-panel')}>
-        <ConnectedManifestForm setLastRequested={this.setLastRequested} />
-        <ul>{manifestList}</ul>
+      <Drawer
+        className={classNames(classes.drawer, ns('workspace-control-panel'))}
+        variant="permanent"
+        classes={{ paper: classNames(classes.drawer) }}
+        open
+      >
+        <Menu id="add-form" anchorEl={anchorEl} open={Boolean(anchorEl)} onClose={this.handleClose}>
```

Comment:
We should use semantic elements and wrap this menu in a <nav> tag, e.g. `<nav id="mirador-workspace-control-panel-menu">`.

Reply:
I'm not sure it's so much of a nav menu right now; it's just wrapping a single form that behaves modal-y; the good news is, the whole thing is temporary until we build out the actual page.